src/main/java/bio/terra/tanagra/query/bigquery/BigQueryExecutor.java

```diff
@@ -24,39 +31,36 @@ public BigQueryExecutor(GoogleBigQuery bigQuery) {
 
   @Override
   public QueryResult execute(QueryRequest queryRequest) {
-    String sql = queryRequest.getSql();
+    String sql = renderSQL(queryRequest.query());
     LOGGER.info("Running SQL against BigQuery: {}", sql);
     TableResult tableResult = bigQuery.queryBigQuery(sql);
 
     Iterable<RowResult> rowResults =
         Iterables.transform(
             tableResult.getValues(),
             (FieldValueList fieldValueList) ->
-                new BigQueryRowResult(fieldValueList, queryRequest.getColumnHeaderSchema()));
+                new BigQueryRowResult(fieldValueList, queryRequest.columnHeaderSchema()));
```

Comment:
How come all of these are changing away from getters? I wouldn't expect to see this change.

Reply:
This renaming is due to changing `QueryRequest` to a `record`. This is one of many changes I made which weren't required but made it easier for me to understand and reason about the code.